Zend/zend_types.h

```diff
@@ -638,6 +638,10 @@ struct _zend_ast_ref {
 #define _IS_BOOL					18
 #define _IS_NUMBER					19
 
+/* used for PFAs/FCCs */
+#define _IS_PLACEHOLDER_ARG         20
+#define _IS_PLACEHOLDER_VARIADIC    21
```

Comment:
Are these required to be a zval type for the following implementation? Other AST attributes use a “distinct” set of values.